template/src/models/models.js

```diff
@@ -1,18 +1,20 @@
 import { File } from '@asyncapi/generator-react-sdk';
 import { TypeScriptGenerator, FormatHelpers, TS_COMMON_PRESET } from '@asyncapi/modelina';
 
+import { Parser } from '@asyncapi/parserV2';
+const parser = new Parser();

```

Comment:
You should also register custom parsers from `@asyncapi/parserV2/cjs/schema-parser/**`

https://github.com/asyncapi/parser-js/tree/next-major/src/schema-parser

Reply:
Right... Effectively it's not possible to use any other input atm..

I don't know, I don't really want to use redefine the entire parser functionality here, also because I don't think it can process local references.

Not sure if you see a better approach then this?